functions/helloworld/test/sample.integration.http.test.js

```diff
@@ -19,8 +19,8 @@ const path = require('path');
 const requestRetry = require('requestretry');
 const uuid = require('uuid');
 
-const PORT = 9010;
-const BASE_URL = `http://localhost:${PORT}`;
+const PORT = process.env.PORT || 8010;
```

Comment:
I prefer 8080 if we are going to change the PORT.

Reply:
The docs and other samples use `8010` - any objection to keeping it consistent?

Reply:
It looks like the docs only has Node right now:

https://cloud.google.com/functions/docs/testing/test-http#integration_tests

Let's use 8080, that is the default of the FF (makes it easier to run locally).

https://github.com/GoogleCloudPlatform/functions-framework#specification-summary

And the point of the sample isn't about customizing ports.

Reply:
Python uses 8080, I think Java might too.

Reply:
Got it. Changed this to `8080`.